.gitignore

```diff
@@ -1,2 +1,3 @@
 *~
 *.pyc
+flycheck_*
```

Comment:
This addition was because I got _real_ tired of my `rg` taking me into files fisketur flycheck created, and adding a `flycheck_*` filter was the easiest way to avoid that. We may or may not want this in.